features/json-raw.yml

```
@@ -0,0 +1,8 @@
+name: JSON source text access
+description: The JSON source text access API is the `JSON.rawJSON` static method, `JSON.isRawJSON()` static method, a source context parameter for the `JSON.parse()` reviver callback, and `rawJSON` value handling in `JSON.stringify()`. You can use this API for lossless serialization.
```

It feels a bit odd to me to start this description by listing the constituent features, and to end it by why you'd use this.

```suggestion
description: The `rawJSON` value handling in `JSON.stringify()` and the source context parameter for the `JSON.parse()` reviver callback serialize and parse JSON in a lossless way.
```

I know this doesn't explicitly mentions the isRawJSON and rawJSON methods, but I don't think feature descriptions should (and do) list all constituent features.
I'm not familiar with this API, but it seems like the entire goal is to stringify and then parse JSON in a lossless way. The technical way to do this deserves longer documentation, which is what MDN is for.

---

I think lossless serialization is one use case but you could do other things with raw JSON access. So I think the idea was to describe the new methods/parameters this feature brings and then to mention one use case of it.

---

Can you let me know what the other use cases are? I'd love to see whether there's a way to summarize/condense them so that it seem like a developer feature, rather than a list of APIs.

---

hm, I haven't found anything. The docs and the proposal just talk about bigints and [escaped strings literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/rawJSON#using_json.rawjson_to_create_escaped_string_literals). So, maybe this is all about lossless serialization after all.

---

@josh-cena, maybe you could help us out here? We're looking for a good description for the new "JSON source text access" proposal.

---

I wrote a long thing but then I decided that @captainbrosset's suggestion is basically my suggestion :D